shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -406,6 +407,13 @@ private int getPointerChangeForAction(int maskedAction) {
         if (maskedAction == MotionEvent.ACTION_CANCEL) {
             return kPointerChangeCancel;
         }
+        if (maskedAction == MotionEvent.ACTION_HOVER_MOVE ||
```

Comment:
Why not add another phase explicitly for the "hover enter" like you did for the "exit"?